src/services/completions.ts

```diff
@@ -1068,10 +1068,12 @@ namespace ts.Completions {
             }
 
             function addSymbol (symbol: Symbol) {
-                if (insertAwait && !symbolToOriginInfoMap[getSymbolId(symbol)]) {
-                    symbolToOriginInfoMap[getSymbolId(symbol)] = { kind: SymbolOriginInfoKind.Promise };
+                if (preferences.includeCompletionsWithInsertText) {

```

Comment:
Shouldn't this be something like `if (!insertAwait || preferences.includeCompletionsWithInsertText)`? Seems like `addSymbol` would now reject _all_ completions if the preference is disabled 🙃